app/src/main/java/edu/grinnell/appdev/grinnelldirectory/LoginActivity.java

```diff
@@ -2,12 +2,29 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 
 public class LoginActivity extends AppCompatActivity {
+    @BindView(R.id.username) EditText username;
```

Comment:
you should use access modifiers here. Also, please remember to use our conventional variable naming. eg. This should read: 
`@BindView(R.id.username) private EditText mUsernameEditText`; or ``@BindView(R.id.username) private EditText mUsername`;

In the end, we want anyone who sees this variable called anywhere to know that it is a member variable (hence the "m" prefix) and should you choose to call it `mUsernameEditText` it also makes us know what it really is (an EditText which is a member variable)